src/model/trust.page.model.ts

```diff
@@ -26,8 +26,31 @@ type TrustWhoIsInvolved = {
   trustName: string;
 };
 
+type TrustHistoricalBeneficialOwnerPage = {
+  id: string;
+  trustName: string;
+  TrustHistoricalBeneficialOwner: TrustHistoricalBeneficialOwner[];
+
+};
+
+type TrustHistoricalBeneficialOwner = {
```

Comment:
this type need populate from appData and pass as formData parameter to template

Reply:
removed